cli/commands/start.go

```diff
@@ -112,7 +112,7 @@ func (cmd *Start) Run(c *cli.Context) error {
 
 	cmd.out.Verbose.Println("Launching Dashboard...")
 	dash := Dashboard{BaseCommand{machine: cmd.machine, out: cmd.out}}
-	dash.LaunchDashboard(cmd.machine)
+	dash.LaunchDashboard(cmd.machine, true)
```

Comment:
My thought was that all containers we use should have an auto-update with the option to suppress.